worlds/gazebo_worlds/igvc_ramp.world

```diff
@@ -113,6 +113,11 @@
               <size>4.5 2 0.1</size>
             </box>
           </geometry>
+          <material>
+            <ambient>0.8 1 0.8 1</ambient>
+            <specular>0.2 0.2 0.2 64</specular>
+            <diffuse>0.4 0.4 0.4 1</diffuse>
+          </material>
```

Comment:
why is the only world you add a material to?

Reply:
It was white before for some reason...

Reply:
so this material is for colouring the ramp correct?

Reply:
yes (it matches the other numbers)